libs/crypto/src/threshsign/bls/relic/CMakeLists.txt

```diff
@@ -0,0 +1,124 @@
+set(INSTALL_DIR "install")
+ExternalProject_Add(relic
+                    PREFIX relic
+                    GIT_REPOSITORY "https://github.com/relic-toolkit/relic"
+                    GIT_TAG "0998bfcb6b00aec85cf8d755d2a70d19ea3051fd"
+                    GIT_PROGRESS TRUE
+                    LOG_DOWNLOAD 1
+                    LOG_BUILD 1
+                    CMAKE_ARGS  -DCOMP=${COMP_FLAGS}
+                                -DALLOC=AUTO 
+                                -DWSIZE=64 
+                                -DWORD=64 
+                                -DRAND=UDEV 
+                                -DSHLIB=ON 
+                                -DSTLIB=ON 
+                                -DSTBIN=OFF 
+                                -DTIMER=HREAL   
+                                -DCHECK=on 
+                                -DVERBS=on 
+                                -DARITH=x64-asm-254 
+                                -DFP_PRIME=254  
+                                -DFP_PMERS=off 
+                                -DFP_QNRES=on 
+                                -DTESTS=0
+                                -DBENCH=0
+                                -DCMAKE_INSTALL_PREFIX:PATH=${INSTALL_DIR}
+                    CMAKE_CACHE_ARGS    
+                                -DFP_METHD:STRING=${FP_METHD_FLAGS}
+                                -DFPX_METHD:STRING=${FPX_METHD_FLAGS}
+                                -DPP_METHD:STRING=${PP_METHD_FLAGS}
+                    BUILD_COMMAND ${CMAKE_MAKE_PROGRAM} -j${NPROC}
+                    INSTALL_COMMAND ""
+                    DEPENDS gmp
+)
+
+ExternalProject_Get_Property(relic BINARY_DIR)
+ExternalProject_Get_Property(relic SOURCE_DIR)
+set(RELIC_STATIC_LIBRARY "${BINARY_DIR}/lib/librelic_s.a")
+set(RELIC_INCLUDE_DIRS "${SOURCE_DIR}/include;${BINARY_DIR}/include")
+message(STATUS "RELIC_STATIC_LIBRARY ${RELIC_STATIC_LIBRARY}")
+message(STATUS "RELIC_INCLUDE_DIRS ${RELIC_INCLUDE_DIRS}")
+
+set(bls_relic_source_files
+        BlsAccumulatorBase.cpp
+        BlsAlmostMultisigAccumulator.cpp
+        BlsAlmostMultisigCoefficients.cpp
+        BlsBatchVerifier.cpp
+        BlsMultisigAccumulator.cpp
+        BlsMultisigKeygen.cpp
+        BlsMultisigVerifier.cpp
+        BlsNumTypes.cpp
+        BlsPolynomial.cpp
+        BlsPublicParameters.cpp
+        BlsThresholdAccumulator.cpp
+        BlsThresholdAccumulatorBaseExports.cpp
+        BlsThresholdFactory.cpp
+        BlsThresholdKeygen.cpp
+        BlsThresholdSigner.cpp
+        BlsThresholdVerifier.cpp
+        FastMultExp.cpp
+        LagrangeInterpolation.cpp
+        Library.cpp
+        )
+
+add_library(bls_relic STATIC ${bls_relic_source_files})
+add_dependencies(bls_relic relic)
+target_include_directories(bls_relic PUBLIC . .. ${RELIC_INCLUDE_DIRS})
+target_link_libraries(bls_relic PUBLIC ${RELIC_STATIC_LIBRARY} ${GMP_STATIC_LIBRARY})
```

Comment:
wasn't this removed in a previous commit?

Reply:
moving threshsign and removing BLS were in separate commits